From 64d60306686218b5958cb52d295807373206f236 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:35:00 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh release-keys.asc from keyservers/WKD: same certificate(s), with
self-signatures our copy was missing; the commands are in the spec
comment.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 lld18.spec       |  12 ++-
 release-keys.asc | 233 +++++++++++++++++++++++++++++++++--------------
 2 files changed, 173 insertions(+), 72 deletions(-)

diff --git a/lld18.spec b/lld18.spec
index b22e4e8..34421f2 100644
--- a/lld18.spec
+++ b/lld18.spec
@@ -49,7 +49,7 @@
 
 Name:		%{pkg_name}
 Version:	%{lld_version}%{?rc_ver:~rc%{rc_ver}}%{?llvm_snapshot_version_suffix:~%{llvm_snapshot_version_suffix}}
-Release:	10%{?dist}
+Release:	11%{?dist}
 Summary:	The LLVM Linker
 
 License:	Apache-2.0 WITH LLVM-exception OR NCSA
@@ -60,6 +60,10 @@ Source0:	%{llvm_snapshot_source_prefix}lld-%{llvm_snapshot_yyyymmdd}.src.tar.xz
 %else
 Source0:	https://github.com/llvm/llvm-project/releases/download/llvmorg-%{maj_ver}.%{min_ver}.%{patch_ver}%{?rc_ver:-rc%{rc_ver}}/%{lld_srcdir}.tar.xz
 Source1:	https://github.com/llvm/llvm-project/releases/download/llvmorg-%{maj_ver}.%{min_ver}.%{patch_ver}%{?rc_ver:-rc%{rc_ver}}/%{lld_srcdir}.tar.xz.sig
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - D574BD5D1D0E98895E3BF90044F2485E45D59042 474E22316ABF4785A88C6E8EA2C794A986419D8A | sq keyring filter --experimental --cert D574BD5D1D0E98895E3BF90044F2485E45D59042 --cert 474E22316ABF4785A88C6E8EA2C794A986419D8A --output update.pgp
+#   sq keyring merge --output release-keys.asc.new release-keys.asc update.pgp
+#   mv release-keys.asc.new release-keys.asc
 Source2:	release-keys.asc
 %endif
 
@@ -87,6 +91,7 @@ BuildRequires:	python3-lit
 
 # For gpg source verification
 BuildRequires:	gnupg2
+BuildRequires:	openpgpverify
 
 Requires(post): /usr/sbin/update-alternatives
 Requires(preun): /usr/sbin/update-alternatives
@@ -117,7 +122,7 @@ Shared libraries for LLD.
 
 %prep
 %if %{without snapshot_build}
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %endif
 
 %autosetup -n %{lld_srcdir} -p2
@@ -218,6 +223,9 @@ export LD_LIBRARY_PATH=%{buildroot}/%{install_libdir}
 %{install_libdir}/liblld*.so.*
 
 %changelog
+* Fri Oct 23 2026 Jakub Jelen <jjelen@redhat.com> - 18.1.8-11
+- Verify upstream source signatures with openpgpverify
+
 * Thu Jul 16 2026 Fedora Release Engineering <releng@fedoraproject.org> - 18.1.8-10
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
diff --git a/release-keys.asc b/release-keys.asc
index 0d3789a..adc2e97 100644
--- a/release-keys.asc
+++ b/release-keys.asc
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQGNBGLtemUBDADClvDIromq0Y4TX+wyRyYCq5WusPQheQuY8dVCXd9KhMpYAv8U
+xsDNBGLtemUBDADClvDIromq0Y4TX+wyRyYCq5WusPQheQuY8dVCXd9KhMpYAv8U
 X15E5boH/quGpJ0ZlVkWcf+1WUHIrQWlbzQdIx514CDM7DBgO92CXsnn86kIMDW+
 9S+Hkn8upbizT1fWritlHwzD9osz7ZQRq7ac03PPgw27tqeIizHGuG4VNLyhbbjA
 w+0VLFSu3r219eevS+lzBIvR5U9W720jFxWxts4UvaGuD6XW1ErcsTvuhgyCKrrs
@@ -8,8 +8,8 @@ gxO5Ma/V7r0+lqRL688ZPr4HxthwsON1YCfpNiMZ6sgxT8rOE0qL/d07ItbnXxz6
 KdcNWIXamTJKJgag6Tl0gYX4KIuUCcivXaRdJtUcFFsveCorkdHkdGNos403XR89
 5u9gq7Ef10Zahsv5GjE2DV5oFCEhXvfIWxvyeJa65iBkJafElb2stgUjkIut2a2u
 +XmpKpwpGSFklce1ABLrmazlLjhsYiJVrz5l5ktoT9moE4GaF7Q5LD6JgsxzLE0U
-Tzo9/AQPd8qG2REAEQEAAbQeVG9iaWFzIEhpZXRhIDx0b2JpYXNAaGlldGEuc2U+
-iQHUBBMBCAA+FiEE1XS9XR0OmIleO/kARPJIXkXVkEIFAmLtemUCGwMFCRLMAwAF
+Tzo9/AQPd8qG2REAEQEAAc0eVG9iaWFzIEhpZXRhIDx0b2JpYXNAaGlldGEuc2U+
+wsEUBBMBCAA+FiEE1XS9XR0OmIleO/kARPJIXkXVkEIFAmLtemUCGwMFCRLMAwAF
 CwkIBwIGFQoJCAsCBBYCAwECHgECF4AACgkQRPJIXkXVkEKoNwv+MEMVzdnzJarx
 ZJ0OzHrGJJG8/chkuoejTjCLG73li9yWQigy5KmvynB5yW0fk0PAQ90vvp2wr/Hd
 mUh0Zda3CwF6sWlO3N6DEDjVA3lZUuofTtvMn/tdGvvOOXYXAP9N+SZfp/7q8dxX
@@ -18,7 +18,7 @@ UIQEqPDE+qOyJ9dRaiL0CUI4x+1wXeXB3OA7VybF2LvaZDbMlocdy+vs825iUWfa
 n8g3mE2TpV8jkc9UHgGGopzxqNquvkkIB7ZFZm/PSW40W3OeHKhYsZZbHrz9403t
 3R4SAzA3ApmMP/P8ue9irsbe24g3rzYMvck1w4C1a4Uy9buT0OCfA+dA16IRAPgV
 5SJEIS62cFbUxkw8el3rUK9V+6kwoq4k8Fs8f1U7DEnOKS/v8BJJCNEc1cvimZai
-Y5/3r5BeneEmuZFKX4iIIfcn5PmLSDB4aw+gKAIAAus+E2DxBqE+uQGNBGLtemUB
+Y5/3r5BeneEmuZFKX4iIIfcn5PmLSDB4aw+gKAIAAus+E2DxBqE+zsDNBGLtemUB
 DADBCNyvUdv0OV//6pQ/0YC2bYXL/ElF0rOjFFl4H7O3TRxgIz2C4nQJHUOrXSmo
 iL7ldfUjoAMgebcoWDpgE8S2Vjw2Gd+UJBQXj+3J6dPKLBUCjj9CLyb5hwOHITMV
 b9UC/E+iwpn4vgTbI6K1O847brkBC+GuDT4g9D3O3sRbja0GjN0n2yZiS8NtRQm1
@@ -27,7 +27,7 @@ R3M5p1nctX15oLOU33nwCWuyjvqkxAeAfJMlkKDKYX25u1R2RmQ4ju2kAbw0PiiZ
 yYft8fGlrwT4/PB3AqfKeSpx8l9Vs15ePvcuJITauo3fhBjJ6Y4WCKlTG1FbDYUl
 KvPhyGO8yLhtZJg3+LbA5M/CEHsDmUh7YEQVxM0RTQMTxNBVBF5IG/4y8v/+19DZ
 89VdpsQF3ThoPV0yh57YMemTBeIxpF9Swp5N7kUWct4872kBnXOmbp/jhU4MpLj6
-iLEAEQEAAYkBvAQYAQgAJhYhBNV0vV0dDpiJXjv5AETySF5F1ZBCBQJi7XplAhsM
+iLEAEQEAAcLA/AQYAQgAJhYhBNV0vV0dDpiJXjv5AETySF5F1ZBCBQJi7XplAhsM
 BQkSzAMAAAoJEETySF5F1ZBCdPwL/3Ox6MwrKFzYJNz3NpQFpKFdDrkwhf25D/Qw
 vu5e8Lql/q62NIhEKH3jxXXgoFYas2G7r8CSCRehraDqvXygbaiWUIkxSU0xuDTl
 lNqHSkCRqIxhi/yxNm1Pk84NVGTLXWW0+CwT9cRwWn5foIPJhoDdZ732zJ7rcY3R
@@ -36,69 +36,162 @@ Ax1gu03o8GD5LFDUuGbESgDJU6nVtVyht7C6AlJWqSX6QS3+lPCw5BOCKbxakYNR
 /oBNauzyDISdbUwzHM2d+XGCjBsXKRA0Tft2NlG6EC83/PuY2J9MSA2gg3iPHsiN
 J5iipbdZNpZ3XL0l8/t/7T60nM7UZDqt3twLMA0eRFRlCnhMjvFE5Zgj5DE7BsJh
 w2nCoGWkAcfeuih+jfyEjN24NK+sE/bM+krwVv430ewJwm1bVUqKrbOb9aa6V9gP
-9RmlwZlOTFGcWBYl/cfRONn9qi9a6w==
-=Lvw+
------END PGP PUBLIC KEY BLOCK-----
------BEGIN PGP PUBLIC KEY BLOCK-----
-
-mQINBFrqgT0BEAC7xo0WH+eNrLlU5LrCk59KmImn1abFcmWNd8kYr5XfqmJKyVqo
-EY7A/yRjf+Yn1621EDkpKPjbql7q7MlZMpqKVdOWKWgmhvz08IOKJxaIABd/iIRT
-FwhIvB68YjtmzcoOJRi1wLnwuG55fJ9E69HyZ33jgAlRaWV3bE/YyszoTlZriUOE
-RbzC5WzX004cE9evlrr+YLt5Y6z7tntOdSXPLyGOFAO5LYMsHsEdi2JBYWrjlslG
-6iJr5iEt9v442PrJ79YYbu5QWe/6APRWtI3AtKBp7y250oon2lbj+bIVD7U9fOBB
-n/Frqx54UN22sJycET63hgYW4pIjIi5zq+FF15aU+ZqBdtNltoX4hEN7wlDpuNc0
-ezVu2Z8hdt8thpjiFUioTQ1t3RmsN6N548VwxmHdoYpAmiZqPIYBYvm85JB7S/3h
-RLuoeGxufBhXGCpnG8ghTOGtbbdanuLB/UROFXTdyZbTCBN5S6jvwkPSaHG7H35Z
-3fazMriTXwL1RGAbKITSWhDe5dXy/yOInWe8emJx+35vwQYCB2L4S8wRyQyRw6x4
-YoXCscW041DUMBX2CC7SjMCcmAC39UX1c3GbTpS3rkJR9cmXt50nviMnKpIwlIPd
-ZYhmxKifwTJ70+c4GVK2o0MG9bTYvpYhLnYxv6iJCfgmT40E+qkDSzSoZwARAQAB
-tCJUb20gU3RlbGxhcmQgPHRzdGVsbGFyQHJlZGhhdC5jb20+iQI/BBMBAgApBQJa
-6oE9AhsDBQkB4TOABwsJCAcDAgEGFQgCCQoLBBYCAwECHgECF4AACgkQoseUqYZB
-nYp8Gg//RmX6Nup/Dv05jTL7dKRBSD08MF400tRtTmRhIuAgGv27qO2hbqzprKVu
-vd20vKBB9CNZpXC2oY8k9VhGv2PZNi/X7iuULIYmzjeFMbJ5CjU6XvuUBmNasITH
-6K/0KLhGebPs5h/DNtd7lbzDm86dLcjxgl6LXUULaSyYvTAKn6YB6mAv5J3qJs2X
-lfTmenNh9p7TPFTfcMHcS70ywjqKXlDiH0q9bRKJnSX7xUFlTHjKkNnAcRjlPaGf
-wUUhIPrnpDboqfwfcmScLrHANW9nwFWSFkNAJu1HQUEuF+An/RZUHDxFbLPKKAIp
-hwZ0aORTfBVZ80AjehDMYCbmp1DJeTyLjC1/94un6mlxPIKnPPPM8rMxr83xnrvP
-+Y1+pJaDUL7ZvKnmt2LrGRa9GvsNiYKpCNCORfiwZTeSxxXb+LgaodnbCHvGBnk7
-nlbLdMY08vNlxSx8LNyG0krFxJw/rq260+73yc+qjENeG68fozTEy/4jSVrF4t3m
-8AAUu5r6i/Aomo7Q27TjU928bbCVunpvDpserfDqr3zsA96LO9k8T6THR6zC9i+R
-LiN9Vjl+Rr2YuU26DjFYkCNEA2kNflYCWPJi5I0eodTPZrIPBWJ+H0YTRX31bMH9
-X88FnWJuCwaqAMN3rWlX/lXNCouWDdCuPWseZApISAMnVDE2mM+JAlYEEwEIAEAC
-GwMHCwkIBwMCAQYVCAIJCgsEFgIDAQIeAQIXgBYhBEdOIjFqv0eFqIxujqLHlKmG
-QZ2KBQJgkytfBQkJaxEiAAoJEKLHlKmGQZ2Kv8YP/jNPjcMAP0ZTpUcYV46mGKwf
-aQ0g5FUMSfxP7uJHtctj2dUckPGpA9SAH+ApiJutVgTQpWqNJKPd2vVxOiu5sywN
-iDKCOMlKug5m6lgLX5h3zBvSN90Hpn4I0qHRA3rgENLoPs/UYBxohvFPIhOOjPqO
-HIUuSPhAIuIZawxtqlADswHiKPy38Ao5GnWRb60zKfrB+N+ZiOtg7ITrlTGYm2tX
-0W9iWUG32gIA/RX2qmFPoLrDFqsk66Eir0Ghk5gppRrmpEl/M1lqA8bxlqWto/8w
-V8yDbSEu5fmM3WN3OUcSA23lYJi4j656Q4hS5PU+IWuZbBhcpYwDGexV5+m/ySZb
-wtHZMIb4Au+dgJHCvRiSqHgplyfiamxX5CfA0DJVHoGXpBOw8a2geRT0+DrjSbOS
-+CDDnlfmQLfHgjEuyQPU8V0Wlb0tJEvnPPqNPmAv0Rv7MC4qmD/zDrgwuddpfr1x
-H+nWus2plR8E6p/x9uvPLb3plJ94ri1XjXiJPyPvqzBAwA40Zeg0rE7sTVwCC3E9
-RZa7dHh17exkcZdOIS/vRQ1G/VNaOVUwrcC/vIMgZSe37bCLeOKViMtacAiBJDjo
-INC1QJ2F3CYVwktrcgmuz9S8e2WrqdTWwijjConB80EwfHQllz5sp/jU6Bgv297X
-UXkgpk1y+ibQ9/syRQpFuQINBFrqgT0BEADB2vDHMuexkaUm3rPi6SvyMGcsHWle
-feAWm+UjROKIaV77rHwo0/8ncKbtoQx4ZZjDXdI77M2bXB7tPgyEML90bWDMMGN/
-gnpwWSsiDRWpFIV/+hD6B+l9UaEi4UvEstUbIchOkGrZgPZ4Qism4FSVosEK+FE7
-EKCT4PSS+LiBKSxJZB8/g2uX+3pJvVxYurvcVpTmiNlXvUyll4KMpzy5e0KKa/0y
-w9h7SAre5399cSM8E7PDQZQDb1EwbyVyO2yDLgs+p3yzPtRJAydaqRPmT1JbSCYf
-hcihTrViMA4EDN5GRjH2EElI37+2HMpgLs4rc6Abz1F4FUVFhqWJXCKUcAIrG17w
-A7YUlYg38S6Xws2Xj1VfZ/WP7/qIMJZidYTHZbN9WWCaifCPfLlE5VDNsa8y6Mxm
-uFMBAB4PpB1gmmP9pPZsOzV9SmeYt8h2P8cVKDW2f56azpBZvZX6NFn8e0+ZDXS4
-8BQz31G2Xdfa3uOEV0J3JxPXcEbfuPzDHb7OMYP+2Ypjox1TozT1e9zr46SQl9OF
-MglOBnwLZJ9baA/IqZkqLq5iu5Oqda44EIVNAntQ3gebi3+q3YG1SvNUseIy2+8y
-cNWtdDuWv366Af0okCdrKAdap8+KbREer9uXhamtvxc49RCoWwuKoKfBz0RdVvMv
-R/Py2xV8A7PaIQARAQABiQIlBBgBAgAPBQJa6oE9AhsMBQkB4TOAAAoJEKLHlKmG
-QZ2KAaMQALHif2E0PBLVt09vlr4i8jAsQvDrzRajmVPd2B9RpfNU6HJe/y93SZd2
-udr9vzgmfd2o5u12vbegKNiMRgp1VyHQDmYlce27jrH5aPuKmos78+o5/p5yPWCv
-Rj8zxGKh7le7UPO+7UveKu+bgb3zwTj6bEuHX7fVI+WjGmEH3bbjDGamWxXrpfGc
-7+Jr8TN4ZO2OwYBcFOS9U2ZQ6TxrPaCSIm6+j8f+a9HPOuuDc62mMuV/EWQZy0i7
-DhDqU2PNpVjQDWQNpHA8oLDrjNFAoJS8gbHABVsFM1VnwBNT2MKcZQmm05dlQ+ll
-S6meHNCvTniKIKC+Giz1Yd5JVGDACZWWPxEz6VhpQW/twkxRqwlUdpFt7UgDquTL
-M1beQUCZRt81yJTNdrggbhQ2POxOdIO0CPiQv7U1IzndZp6baedeBw4a7FCbj6GY
-cQeHxQCrWpQrwigiseG5uhhS9aiaVFEHja9baSLfXlZu/vsR4MdDG5/iEpier/Xw
-h1qnpTSY+r31Uw3lTUlPHzlg47PMgPslaIhCzfVggxh9bTqxcDbuYJ7NuoMho3tN
-yWfeofTJ7PhKzoXM2Y/rRFoM5gNh1RVA19ngLT5Jwiof8fPZvHJ/9ZkHn+O7eMNm
-m5++gYza3pnn2/PoGpGGAKok+sfJiq5Tb7RUefyJTeZiyTZ/XJrA
-=tMzl
+9RmlwZlOTFGcWBYl/cfRONn9qi9a68bBTQRa6oE9ARAAu8aNFh/njay5VOS6wpOf
+SpiJp9WmxXJljXfJGK+V36piSslaqBGOwP8kY3/mJ9ettRA5KSj426pe6uzJWTKa
+ilXTliloJob89PCDiicWiAAXf4iEUxcISLwevGI7Zs3KDiUYtcC58LhueXyfROvR
+8md944AJUWlld2xP2MrM6E5Wa4lDhEW8wuVs19NOHBPXr5a6/mC7eWOs+7Z7TnUl
+zy8hjhQDuS2DLB7BHYtiQWFq45bJRuoia+YhLfb+ONj6ye/WGG7uUFnv+gD0VrSN
+wLSgae8tudKKJ9pW4/myFQ+1PXzgQZ/xa6seeFDdtrCcnBE+t4YGFuKSIyIuc6vh
+RdeWlPmagXbTZbaF+IRDe8JQ6bjXNHs1btmfIXbfLYaY4hVIqE0Nbd0ZrDejeePF
+cMZh3aGKQJomajyGAWL5vOSQe0v94US7qHhsbnwYVxgqZxvIIUzhrW23Wp7iwf1E
+ThV03cmW0wgTeUuo78JD0mhxux9+Wd32szK4k18C9URgGyiE0loQ3uXV8v8jiJ1n
+vHpicft+b8EGAgdi+EvMEckMkcOseGKFwrHFtONQ1DAV9ggu0ozAnJgAt/VF9XNx
+m06Ut65CUfXJl7edJ74jJyqSMJSD3WWIZsSon8Eye9PnOBlStqNDBvW02L6WIS52
+Mb+oiQn4Jk+NBPqpA0s0qGcAEQEAAc0iVG9tIFN0ZWxsYXJkIDx0c3RlbGxhckBy
+ZWRoYXQuY29tPsLBlgQTAQgAQAIbAwcLCQgHAwIBBhUIAgkKCwQWAgMBAh4BAheA
+FiEER04iMWq/R4WojG6OoseUqYZBnYoFAmgafDMFCQ8RLnYACgkQoseUqYZBnYr/
+gQ/9GkPts+iT8vIzLOBBnzdOAbk9dllu5Hla8l93t53d0Z6XHnEZ67JOqulb5YQZ
+ipH56527cU9csUBFrnYnSlNasXpm3rvgRtvipp4AHpKgyUWHRTuaa2qV38lxKAD8
+NfLzF+JEEwcuAvqbzXtkKyG2ME2Th+HYXJjqqr80Q7W5DQnTF30YLLEjXpA3e3Cr
+fhjx5StM7gWECL1quicO00VHwMgXU6Xta+hfIKe948lDOZ9e0b4lzNyoTEhwmhBq
+KhI4uKOxrT7UZrUTxvFZDJP751Xg1dCxO4FpY7m9WTGIerefdORkJx3cKu9mXXVY
+7SHpRnKVi3kBHL1FxSKxP+mioaAYVlFpmg8o+Nk8jhCl0SHYCfqmd1TmwtNZPUI6
+9hZhxPnnyfn7RMHFmwqji6hGj9Up0kCXWBPBTYDWS0J229BFJnlc0k5wzd7vxEpQ
+Yfc1QZeZLBeVdnttrm1dxI8efzFoMCN5RAjo6Xs51yChVfjfu5HEOI+sOi+DNbcl
+UbmU2wly1zcHC906r9zdAaoxjMMx6YzWx8fLvaPyyXTDJlGl1lyAKibi6Lwp+XpQ
+npOK4N2YERglFO6oKlzxE0yyasx5IYVKfHpa89UeicZphizRHB1kqOCD2Q/H41eb
+n9XmTMG87iVS4veu9KRJIEIw1icubNP7/TdBC4K5S3okghXCwZYEEwEIAEACGwMH
+CwkIBwMCAQYVCAIJCgsEFgIDAQIeAQIXgBYhBEdOIjFqv0eFqIxujqLHlKmGQZ2K
+BQJmM9JHBQkNKoSKAAoJEKLHlKmGQZ2KxPIQAJQ/81K1HgUdF4ccNypnkgPmXDrB
+oqwWzwnUZmlb2pVJS52k9UZW6tSV+3ulF8DwuT1R9qeOaerVClf1a9APyyThFind
+u7UOdfsvYZXnRymLc52wNAC7PXtwR5G5WqHg8mOX2CQaZmgQweQS5rBqHG8dnca1
+dZaq6+e3jU/Yoeu+h4pcdJmeur5OUnqrLgQOV7bbETKFR7NEBe1m57c2EN/+pAAv
+1nhynteb7C+opNwgOeAFJurXVVJ9ucD8Ut4sAozSxhcNs7hXW7Pyk4qVykjWEelB
+JvlPfR51cThFtpREyZCt7M2gGkToTd8bqsWotbZW6GQRlwI5c4/qy/uRW8tsQHUU
+fOT4piV6YZcBLe0M/sGOwqCkwwLWijBTX49az6ykWsz/nV8obDp5c02g/CWmf68w
+YcaNNZ4PnS4LAKYPPi5mwnfVh0ZYQkZYN1LSbgi6OCgtvmf/A9Vz/o3h4cCYY/C6
+at7bHErubAM5cfnvcHs8iVSll+FXpjBgelCRAijfva1ZhAh/lKOATw6G/o36zhpK
+LrABqdGfNJ+7r0QHUTyYNvKz8qv+m8i1RPWZ2AcT2y3RFiPPjRKDg5mgMUzryN7b
+Vm93BrUDQ9q3/u+9Je6IsPR4INQOnwKXfGM/GCJOsmtGoP71MnJiP8n4s7F7Ok3r
+LPd537JSfw25ZCZKwsGWBBMBCABAAhsDBwsJCAcDAgEGFQgCCQoLBBYCAwECHgEC
+F4AWIQRHTiIxar9HhaiMbo6ix5SphkGdigUCZGRheQUJC1sTvAAKCRCix5SphkGd
+iubwD/46F6c+byxzx200tWA+Vbll/qaTdoqeuWCzvIUo677oBz2VrbsRDQSpdgc1
+hlmshfF64M4Wt+5LD/9Mv5QvTBsXhWj3va3GGgxbne/rR+f4Abu6xFh2NilTmMnm
+yVv0hII2QJ2HHh6ML7otmTZgACWm4DBfYo8KCGIbIGsjV9b1Ep8fb4nGFO2w9/i+
+CCk9Te5yO7ojEgxQ4FlZ8tlWlG8oQTtHg4xu5MEHWpfQuxjLfxmskWPmzNTHRzCI
+dySRhDpb2u5udcM7xwa8OK0zTMEBS3hoiMCRDp7AUGXFaxFFUc6/jwXUAwhSqH1N
+PpBwH7r8//vWUPyDSzGxHmSpOm+LhKjObIj/LrrVfB058Yw9Izd+6nn28MRtrisf
+6dJWz59wfcdox5F6VYhIXfZkLs0yBWMxg/qILw6SANTddQ4+IqUivxpux16X0oO2
+cof3NutG6RKgjKBRYCxGvjpQe28ezJdMw3o9Y5jzNLTLZxlzFuaSp73LSNfWmJrJ
+xHYeBOynNKmoWiQ1/NzMnQ4dPuUMP/sPrrl5U2u3IGPZ3FlGqBm2iFVIrBbQX5PQ
+uod59yI08ZbOue5w8mYKCOHiuw9JFl14SaIuRFZA9ItcTft/vEOJhxGIgopQ7isQ
+UfWbm2AbovSc/LGjsCS81WQth/VmkUMVlIjtacEs96Q80ASBycLBlgQTAQgAQAIb
+AwcLCQgHAwIBBhUIAgkKCwQWAgMBAh4BAheAFiEER04iMWq/R4WojG6OoseUqYZB
+nYoFAmCTK18FCQlrESIACgkQoseUqYZBnYq/xg/+M0+NwwA/RlOlRxhXjqYYrB9p
+DSDkVQxJ/E/u4ke1y2PZ1RyQ8akD1IAf4CmIm61WBNClao0ko93a9XE6K7mzLA2I
+MoI4yUq6DmbqWAtfmHfMG9I33QemfgjSodEDeuAQ0ug+z9RgHGiG8U8iE46M+o4c
+hS5I+EAi4hlrDG2qUAOzAeIo/LfwCjkadZFvrTMp+sH435mI62DshOuVMZiba1fR
+b2JZQbfaAgD9FfaqYU+gusMWqyTroSKvQaGTmCmlGuakSX8zWWoDxvGWpa2j/zBX
+zINtIS7l+YzdY3c5RxIDbeVgmLiPrnpDiFLk9T4ha5lsGFyljAMZ7FXn6b/JJlvC
+0dkwhvgC752AkcK9GJKoeCmXJ+JqbFfkJ8DQMlUegZekE7DxraB5FPT4OuNJs5L4
+IMOeV+ZAt8eCMS7JA9TxXRaVvS0kS+c8+o0+YC/RG/swLiqYP/MOuDC512l+vXEf
+6da6zamVHwTqn/H2688tvemUn3iuLVeNeIk/I++rMEDADjRl6DSsTuxNXAILcT1F
+lrt0eHXt7GRxl04hL+9FDUb9U1o5VTCtwL+8gyBlJ7ftsIt44pWIy1pwCIEkOOgg
+0LVAnYXcJhXCS2tyCa7P1Lx7Zaup1NbCKOMKicHzQTB8dCWXPmyn+NToGC/b3tdR
+eSCmTXL6JtD3+zJFCkXCwX8EEwECACkCGwMHCwkIBwMCAQYVCAIJCgsEFgIDAQIe
+AQIXgAUCXMtvQwUJBaNVBgAKCRCix5SphkGdiv0BEACqxA5ROcuFOrRn5dcjC2Co
+8dDg3jOVKHTFOI4UvA8zowFSxiLbAUjEnhzXjlH1K5LJNzCP94GqGc6Vp2cFdd99
+FitrGxAfiq58WSuqrB2qkjWgilB0MeA4ovmBiMp7avy3KQtkvOhspIPQqxdPQbb4
+P+gTDf62L/5GCB4u96brkkjciRuoOzdbMarB6IrSa65Ny+Co1JmKlcEvTLEnRRoI
+9K0O9Xr5Xd7+Q97QcQZRtjjtjb5MKio4IA0KBDQc2FoDVdH0/7cn4JH/75j303ml
+bbxm9Dr0QUjHx0dtJZMvRs2GYLTjymRcNZwvTFnrZmZcabRFRzPDCn4jwP9UKUxN
+VMBnfxKw6OqnB9HQt4ecunVojA4/5FdfWNnh8MBsBnaYJgtoH4PBMDdgEyblaYlw
+HnXdETGzAsNNJPSkN2F89CDAUME8iodF0vQ9DbAQgOpxigYCleV0v109sus6NpKI
+Sa8x8jnhthGDlCwtEOOTR8YgUvDWqQBWhiNIuu0BPr+rPXlORdqWnYFoh1WphQ+b
+AHM9lH6yb7f7/xxtKpkoW7VCEueJy8/cwC7My9vF2G8b+snKepHVG7pYlq4ffo0X
+kPBPz9J1cBIqKMweQ78xLhk8mDdJ+LPwVf7n+mlwI27zbWuwSPhwMHZlOiyUulT2
++/LlzgaQueIszpiTxJuyx8LBfwQTAQIAKQUCWuqBPQIbAwUJAeEzgAcLCQgHAwIB
+BhUIAgkKCwQWAgMBAh4BAheAAAoJEKLHlKmGQZ2KfBoP/0Zl+jbqfw79OY0y+3Sk
+QUg9PDBeNNLUbU5kYSLgIBr9u6jtoW6s6aylbr3dtLygQfQjWaVwtqGPJPVYRr9j
+2TYv1+4rlCyGJs43hTGyeQo1Ol77lAZjWrCEx+iv9Ci4Rnmz7OYfwzbXe5W8w5vO
+nS3I8YJei11FC2ksmL0wCp+mAepgL+Sd6ibNl5X05npzYfae0zxU33DB3Eu9MsI6
+il5Q4h9KvW0SiZ0l+8VBZUx4ypDZwHEY5T2hn8FFISD656Q26Kn8H3JknC6xwDVv
+Z8BVkhZDQCbtR0FBLhfgJ/0WVBw8RWyzyigCKYcGdGjkU3wVWfNAI3oQzGAm5qdQ
+yXk8i4wtf/eLp+ppcTyCpzzzzPKzMa/N8Z67z/mNfqSWg1C+2byp5rdi6xkWvRr7
+DYmCqQjQjkX4sGU3kscV2/i4GqHZ2wh7xgZ5O55Wy3TGNPLzZcUsfCzchtJKxcSc
+P66tutPu98nPqoxDXhuvH6M0xMv+I0laxeLd5vAAFLua+ovwKJqO0Nu041PdvG2w
+lbp6bw6bHq3w6q987APeizvZPE+kx0eswvYvkS4jfVY5fka9mLlNug4xWJAjRANp
+DX5WAljyYuSNHqHUz2ayDwVifh9GE0V99WzB/V/PBZ1ibgsGqgDDd61pV/5VzQqL
+lg3Qrj1rHmQKSEgDJ1QxNpjPzsFNBFrqgT0BEADB2vDHMuexkaUm3rPi6SvyMGcs
+HWlefeAWm+UjROKIaV77rHwo0/8ncKbtoQx4ZZjDXdI77M2bXB7tPgyEML90bWDM
+MGN/gnpwWSsiDRWpFIV/+hD6B+l9UaEi4UvEstUbIchOkGrZgPZ4Qism4FSVosEK
++FE7EKCT4PSS+LiBKSxJZB8/g2uX+3pJvVxYurvcVpTmiNlXvUyll4KMpzy5e0KK
+a/0yw9h7SAre5399cSM8E7PDQZQDb1EwbyVyO2yDLgs+p3yzPtRJAydaqRPmT1Jb
+SCYfhcihTrViMA4EDN5GRjH2EElI37+2HMpgLs4rc6Abz1F4FUVFhqWJXCKUcAIr
+G17wA7YUlYg38S6Xws2Xj1VfZ/WP7/qIMJZidYTHZbN9WWCaifCPfLlE5VDNsa8y
+6MxmuFMBAB4PpB1gmmP9pPZsOzV9SmeYt8h2P8cVKDW2f56azpBZvZX6NFn8e0+Z
+DXS48BQz31G2Xdfa3uOEV0J3JxPXcEbfuPzDHb7OMYP+2Ypjox1TozT1e9zr46SQ
+l9OFMglOBnwLZJ9baA/IqZkqLq5iu5Oqda44EIVNAntQ3gebi3+q3YG1SvNUseIy
+2+8ycNWtdDuWv366Af0okCdrKAdap8+KbREer9uXhamtvxc49RCoWwuKoKfBz0Rd
+VvMvR/Py2xV8A7PaIQARAQABwsF8BBgBCAAmAhsMFiEER04iMWq/R4WojG6OoseU
+qYZBnYoFAmgaiYIFCQ8RO8UACgkQoseUqYZBnYq5dg//dNz/7hYjYa7gg9KGQnNq
+QouODbrT9j6QoIjLL3jt8XFokv0qs62v/TJe6Dv2HNhp9gdewUlzvJ9NrXOi9BNB
+3wCXwz/ttNgx++MBWhFQvoAxjov9b/7vahmIzqcN1IQMs0FSnVIZxo+fiAzQcBjj
+6vVPyYN0HiBYUiaDEJfNTacXEzZj4q8tgHwsPPuM3n0AYKURUWfCkd6sU0aN6EWB
+kUZLWr/pGI/76DzBoMLyqkLbChdbEpKPIwnJ6tJtW6Bdv5MbF6f/lIVridOaSwHx
+QRbb7S2RBjoDSsA5cr43S7yqz3iAyXspaXpjCydJtPkRq2sVTf3yUtyQqAF7U227
+AEdGfX0UMrCfgdbNrbGn2ci3Rmc9okJafHDD3KIYcstfHnwRDxE/oJE05W3IXXt4
+veRgUxybUt0TI96FymsuBzTPtvS9jFMNPctshABOJeEMeKZr+yZX2dcTYSTfhDN8
+WHkDSZwHBa6WzdhsZYDeR1G8wYxpcJHU+wGwn4K6/ioxN7/nhkVMyeloxTDDSJkJ
+tP7uBJcjyzAwj+MmtPW7jFAeEkio0HwWCRmX5496wSvZzZEL6/4xJpG7GyG3mMAO
+eCz/KtcX79VszXafS3Y7ILUKQZttbMZ1xTlOmlI5+Tac+SeE39HfWKP35TrOvgFm
+N5wpPmvHN98WtDGbrae6ZmbCwXwEGAEIACYCGwwWIQRHTiIxar9HhaiMbo6ix5Sp
+hkGdigUCZjPSpAUJDSqE5wAKCRCix5SphkGdimxrD/9mG250iX9mU69g34LzsPmf
+xS9Sn6MIEG2pPUTL9a3+hSlb1yXxPexwlbo/99sjrDTon6YVkr+fXcVlxbfrDOPY
+0yQqarJAK6SENIp2YT/y+zSXFsHNz5aRUNy5B7vA1oIOldXG1aae+GFlbeRHMbI9
+l1w0qSbUJmqJwPmtuyt8XymWLf7bHLLIdG+GrS6RgnIoD1bVB4oWqj+USTWU+9A1
+vW/rGYm2/DiZIlFR0uMTRPW+l5U4VBYIdyV8/zeHltN8Hc0guYRRgaHBwe2KZf77
+GkUHL/EFaFcwXDxsLyeQ09v81pQPXF31SJgrN20dyPyoFvbHcoUZlZsqj/1Y18Z+
+F2ZDAxskbByjH6EPs/UNuT+/bbF0hk16dsqJdYftyt1XASTlhLnpxltqpnhiy3NX
+IX8J0vjchp4DKCQJEFl/P+oIS6cyn0eYU4RiDT2dhLA4MUvh27Wpb3wFZqSk+Ig9
+qX2LSpAgleDYIH5sRqbUwVT1HPwXdyIIcUpamTpJHtUeVchnx5xyULmr9IhyNUqS
+1lI4cVA3OaVuUYioyGonQnOFb1tPor7XOwEsKMffsQnkn+fEY0IDDqTh38wxhmjH
+NRxdDFtBx4a/fiB1DlmvKKrNRjgp/5anEti/V3ULxeA2F43zevpQyV41J/nOvUx2
+GICkaACw6T+r86WFapyuU8LBfAQYAQgAJgIbDBYhBEdOIjFqv0eFqIxujqLHlKmG
+QZ2KBQJloup1BQkLVp04AAoJEKLHlKmGQZ2KKXMP/0/V3U5xFittSfgQWw6rbcfc
+F/TVSUBD7Ch88cENOAjkL963RLT1dhZtLY8+U6TqWtRG6xlWuEaLlcU4FyDdVP8o
+ut2ONB3cwh60u7UjP8PRHxLilYkEsqqB/WCHzCQj40job+UfUcR3Lr4M1ewuCbzn
+GuSB+fxE4WmrkANJL05BR6en7o3cxRkoE/2ZaHFxUuWjwA4ZGj62GZVEXf/0tqzw
+SGxMZYAw+pn84OCWXDQF1UoPbRkuLoL1FBkXVzVq3ijtKqxjmCIVdfPksSF7D+mU
+6gm/CdS3tz9l2H6xVDVCdeFMRNvTwaLs4hfNuLMh0xYW9fVJegA2AMsiUGpDSDdz
+O6xp4is42qXeDXpHqYxBBAgs88jJk0QYvzRBeqY+kXXTgC+b/KTAkZo8DcAEzSoS
+eIta07bXsYn7W1dpS6v4nBVdOaCV6pJKhaMlOAvEckfjXwRwjo3i5EACE6zGjj7T
+XgRs1Ka7iK0UH/tvOM5uGK7tMTRavDeENCvSWa7xD76rpHX7/kNX3I52ENj/p/rP
+CgPoH5u2gUAtUnDCa3w8NBtwL5FFzHmpNWGFduJSoeIgh4WU999JZZ9rCxJ70KlI
+kRTvzhUGfJG2O2MN8zu5Hw643vkyqdIJQ8fMHPfox2iIO6KHp0L/Brt6xg5Vq60u
+ENNV427ABJO5li7+DC0gwsF8BBgBCAAmAhsMFiEER04iMWq/R4WojG6OoseUqYZB
+nYoFAmQ+85QFCQlq29cACgkQoseUqYZBnYrKOw//QXJcTHOxvpYR7Qy20r/Vvchw
+wTS1traA8xXf4VbEyL1ao3L+jc3G0ddpusipf/B/W9P8lGDTBZaFKUaQPcWTQTA5
+vK0FnIyiOQtg7407KMmhhxEm9cCU5IHRC9cZGKNQdwguXW3765NA1Vt2/lJnmvRj
+vp3B+X8i6KHt3CcpPmUNzPKBxVqa5zvR5AEI28iviLHJfPpyzud2pcACmeG2w5T1
+Xl/h8DCsiPFYmOWsNxFpqVvWwOviuaVGIuIL3hvJfbkMhPM78i14CAXtKe/Yk8ub
+PD3+SMaWyVU9ndQVXaEYnOAiwgEupmfCNs1XKiZbskSrJrZZqBCchzCs3rvbJp0a
+0JzmBuxOgpNlDNkRr8NHvzDu3xyJkuWEghPUzJQODhhkUIAo0vlT3O0LcqobpByr
+nU3yp0l9qDM4L/UNEpNrRQMz+0t1cKD0YTJCZkO7kp9rSZZwgCfhavTj6cAyLefS
+QlzQLbplSJpP0rk+NxuCeWmfs9S9M+syO0c1n4q+5DpOtNqOCTs5L9t2nbZzCslA
+StlcjGXNsPJSR1DU70Ucq8tV7IifExKKH56MnA6c3Drh6xb06QcreXHpoAmgcEuy
+UcoGeQYSt/1CUrcKnn7dopC1KgrrF9YhBitZsS1SBQx6yxYi6+DnrssUzltqSIAj
+zK1Zatzel6xPvviuukLCwWUEGAECAA8FAlrqgT0CGwwFCQHhM4AACgkQoseUqYZB
+nYoBoxAAseJ/YTQ8EtW3T2+WviLyMCxC8OvNFqOZU93YH1Gl81Tocl7/L3dJl3a5
+2v2/OCZ93ajm7Xa9t6Ao2IxGCnVXIdAOZiVx7buOsflo+4qaizvz6jn+nnI9YK9G
+PzPEYqHuV7tQ877tS94q75uBvfPBOPpsS4dft9Uj5aMaYQfdtuMMZqZbFeul8Zzv
+4mvxM3hk7Y7BgFwU5L1TZlDpPGs9oJIibr6Px/5r0c8664NzraYy5X8RZBnLSLsO
+EOpTY82lWNANZA2kcDygsOuM0UCglLyBscAFWwUzVWfAE1PYwpxlCabTl2VD6WVL
+qZ4c0K9OeIogoL4aLPVh3klUYMAJlZY/ETPpWGlBb+3CTFGrCVR2kW3tSAOq5Msz
+Vt5BQJlG3zXIlM12uCBuFDY87E50g7QI+JC/tTUjOd1mnptp514HDhrsUJuPoZhx
+B4fFAKtalCvCKCKx4bm6GFL1qJpUUQeNr1tpIt9eVm7++xHgx0Mbn+ISmJ6v9fCH
+WqelNJj6vfVTDeVNSU8fOWDjs8yA+yVoiELN9WCDGH1tOrFwNu5gns26gyGje03J
+Z96h9Mns+ErOhczZj+tEWgzmA2HVFUDX2eAtPknCKh/x89m8cn/1mQef47t4w2ab
+n76BjNremefb8+gakYYAqiT6x8mKrlNvtFR5/IlN5mLJNn9cmsA=
+=gogj
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0

